tests/integration_tests/test_server.py

```diff
@@ -175,15 +180,12 @@ def test_server_integration_with_images(csv_filename):
     model_output = model_output.to_dict("split")
     assert model_output == server_response
 
-    # Cleanup
-    shutil.rmtree(output_dir, ignore_errors=True)
-    shutil.rmtree(image_dest_folder, ignore_errors=True)
-
 
 @pytest.mark.parametrize("single_record", [False, True])
 def test_server_integration_with_audio(single_record, csv_filename):
+    tmpdir = os.path.dirname(csv_filename)
```

Comment:
Another option here is to replace `csv_filename` with `tmpdir` and then let `csv_filename = os.path.join(tmpdir, 'dataset.csv`). Would be consistent with other parts of the code, but not a big deal.

Reply:
Done.